src/pages/editProfile.tsx

```diff
@@ -153,101 +204,107 @@ export default function Home() {
+          <Stack
+            spacing={2}
+            sx={{ marginTop: "-40px" }}
+            alignItems="center"
+            justifyContent="center"
+          >
+            <IconButton
+              onClick={handleCancelImageChip}
+              color="secondary"
+              style={{
+                padding: 0,
+                backgroundColor: "black",
+                position: "relative",
+                top: "70px",
+                left: "80px",
+                zIndex: "1",
+              }}
+            >
+              <CloseIcon color={showImageUploadError.err ? "error" : "primary"} fontSize="medium" />
+            </IconButton>
+
+            <Avatar alt="Anya" sx={EditProfileStyle.Avatar}>
+              {input.image && (
+                <Image
+                  src={input.image}
+                  alt="Upload avatar"
+                  width={200}
+                  height={200}
+                  style={!isImageUpload ? { opacity: "0.5" } : { objectFit: "cover" }}
+                />
+              )}
+              <IconButton
+                sx={EditProfileStyle.DeletePictureButton}
+                aria-label="upload picture"
+                component="label"
+              >
+                <input onChange={handleImageChange} hidden accept="image/*" type="file" />
+                <CameraAltIcon sx={{ fontSize: "100px" }} />
+              </IconButton>
+            </Avatar>
+            {showImageUploadError.err && (
+              <Typography align="center" variant="body1" color="error">
+                {showImageUploadError.msg}
+              </Typography>
+            )}
+
+            <Box style={EditProfileStyle.TextField}>
+              <CommonTextField
+                header="Username"
+                icon={ICONS.EDIT}
+                placeholder="Display Name"
+                value={input.displayName}
+                handleValueChange={handleDisplayNameChange}
+                char_limit={CHAR_LIMIT.MAX_DISPLAY_NAME}
+                isErr={isPressSubmit && displayNameErr.err}
+                errMsg={displayNameErr.msg}
+              />
+
+              <DescriptionTextField
+                name="description"
+                header="Description"
+                placeholder="Tell Us More About Yourself!"
+                value={input.description}
+                handleValueChange={handleDescChange}
+                char_limit={CHAR_LIMIT.MAX_DESCRIPTION}
+                isErr={isPressSubmit && descriptionErr.err}
+                errMsg={descriptionErr.msg}
+              />
+
+              <Box style={EditProfileStyle.HalfTextField}>
+                <CommonDropdown
+                  header="Gender"
+                  value={input.gender}
+                  handleValueChange={handleSelectChange}
+                  items={Object.values(GENDER)}
+                />
+              </Box>
+            </Box>
+
+            <NormalButton label="Save" onClick={editProfileBtnOnClick} />
```

Comment:
When I click `Save` button (with all field values are valid), the error appears like this.
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/62459518/226540714-c9dacf28-3ee3-4d2f-a51f-fb855280c3da.png">
(maybe path fault? since it appears `undefine`)

Reply:
also, when I click the `X` button to quit edit profile and press `Quit` button on the popup, It also appear error like this.
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/62459518/226541376-41abce46-2a15-4203-9cb8-29e380caa635.png">

Reply:
Solved